### PR TITLE
refactor(core): Simplify webhook pubsub message handler

### PR DESCRIPTION
**Pull request by ivov**
Simplfy pubsub message handling in a `webhook` process using typed event emitter, add tests, and delete `handleCommandMessageWebhook` and `OrchestrationHandlerWebhookService`. 

https://linear.app/n8n/issue/PAY-2023

Ref: https://github.com/n8n-io/n8n/pull/10566

### CODE REVIEW

File: packages/cli/src/commands/webhook.ts [hunk 1]
```diff
@@ -110,6 +111,11 @@ export class Webhook extends BaseCommand {
 
 	async initOrchestration() {
 		await Container.get(OrchestrationWebhookService).init();
-		await Container.get(OrchestrationHandlerWebhookService).init();
+
+		const subscriber = Container.get(Subscriber);
+		await subscriber.subscribe('n8n.commands');
+		subscriber.setCommandMessageHandler();
+
+		Container.get(PubSubHandler).init();
```
Comment (despairblue) [inline packages/cli/src/commands/webhook.ts:119]:
```suggestion
		Container.get(PubSubHandler).init();

		const subscriber = Container.get(Subscriber);
		subscriber.setCommandMessageHandler();
		await subscriber.subscribe('n8n.commands');
```

Would this order also work and prevent use form missing messages?

**Review by despairblue — APPROVED** [review]

File: packages/cli/src/commands/webhook.ts (hunk 1 above)
Comment (ivov) [reply packages/cli/src/commands/webhook.ts:119]: Thanks! It's a tiny window - I'll address in the next PR 👍🏻

File: packages/cli/src/scaling/pubsub/pubsub-handler.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+				await handlerFn(event);
+			});
+		}
+	}
+
+	// #region Webhook process
+
+	private setupWebhookHandlers() {
+		this.setupHandlers({
+			'reload-license': async () => await this.license.reload(),
```
[the remark this answers is not in the document]
Comment (ivov) [reply packages/cli/src/scaling/pubsub/pubsub-handler.ts:47]: Discussed during call

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Pull request merged**